docs/Plugins/tapd.md

```diff
@@ -8,6 +8,9 @@ description: >
 
 This plugin collects TAPD data through its REST APIs. TAPD is an issue-tracking tool similar to Jira.
 
+## Supported Versions
+Available for all versions
```

Comment:
Please add the "check this doc for more details" sentence to all the manuals. I noticed that some manuals are not added.

Reply:
Okay, sure.